src/controller/java/BUILD.gn

```diff
@@ -76,6 +76,7 @@ shared_library("jni") {
   deps = [
     "${chip_root}/src/app/icd/client:handler",
     "${chip_root}/src/app/icd/client:manager",
+    "${chip_root}/src/controller/data_model",
```

Comment:
why do you need this line? could you show me the error without this line?

Reply:
[build_data_model_error.log](https://github.com/project-chip/connectedhomeip/files/14755942/build_data_model_error.log)
Here is a file with build logs.